science/netcdf-fortran/Portfile

```diff
@@ -11,8 +11,7 @@ PortGroup                   github 1.0
 # hdf5+cxx (w/o a fortran variant) and netcdf-fortran.
 #mpi.enforce_variant         hdf5
 
-github.setup                Unidata netcdf-fortran 4.4.4 v
-revision                    3
+github.setup                Unidata netcdf-fortran 4.4.5
```

Comment:
I did not check the details, but why did you remove the `v` prefix? As a consequence the port fetch seems to be failing.

The maintainer only revbumped the port in d7ca35082ea58d3381d43184ab8ae0525b8edc5a, but maybe he didn't notice the update, or maybe he had a valid reason to hold back. I would suggest fixing the conflict (rebase against master and force-push). I don't know why @tenomoto did not respond yet as he was working on the updates very soon after you submitted the PR.